db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,33 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
+
+    say_with_time("Migrating Kubernetes provider options") do
+      ExtManagementSystem.where(:type => "ManageIQ::Providers::Kubernetes::ContainerManager").each do |cp|
+        migrated_options = {}
+        migrated_options[:image_inspector_repository] = Settings.ems.ems_kubernetes.image_inspector_repository
+        migrated_options[:image_inspector_registry] = Settings.ems.ems_kubernetes.image_inspector_registry
+        CustomAttribute.where(
+          :resource_type => cp.type,
+          :resource_id => cp.id,
+          :section => ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::ATTRIBUTE_SECTION,
+          :name    => ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::PROXY_ENV_VARIABLES).each do |ca|
```

Comment:
@enoodle I don't think we want to reply on the Kubernetes module dependency here for this migration. cc @blomquisg